### DISCUSSION TITLE
What does the FundMe.sol contract even do?

### DISCUSSION THREAD

**Discussion by ArcusConsulting** [smartcontractkit/full-blockchain-solidity-course-py#321] [Q&A] [answered]
Sorry for the question, but I haven't been able to dedicate as much time as I wanted to on this. To the point where I forget the purpose of this FundMe.sol contract (Lesson 5). So what dos this contract even do or what is its purpose?

Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @ArcusConsulting,  FundMe.sol contract is to simulate a crowdfunding system, where people can send ether to a specific contract address. However, the real goal is to teach you how to make a contract payable and how the oracles like Chainlink works.

**Reply by ArcusConsulting** [reply]
Thanks! Looks like I'll have to supplement this course with a lot of reading and other resources to fully grasp this.

**Reply by cromewar** [reply]
Feel free to make any another question.
